x-pack/plugins/security_solution/server/utils/with_security_span.ts

```diff
@@ -35,3 +35,16 @@ export const withSecuritySpan = <T>(
     },
     cb
   );
+
+export const withSyncSecuritySpan = <T>(name: string, fn: (span: Span | null) => T): T => {
```

Comment:
Nit: for consistency with Node idioms

```suggestion
export const withSecuritySpanSync = <T>(name: string, fn: (span: Span | null) => T): T => {
```